website/cue/reference/components/sinks/base/aws_s3.cue

```diff
@@ -602,7 +602,28 @@ base: components: sinks: aws_s3: configuration: {
 		type: object: options: "*": {
 			description: "Tags for a metric series."
 			required:    true
-			type: string: syntax: "literal"
+			type: object: options: {
```

Comment:
I'm pretty sure this is not what we want to show to users, but this file also isn't being used yet so just bringing it up-to-date for now. Separately we should adjust this. cc/ @bruceg